suite/tests/client-interface/events.dll.c

```diff
@@ -168,9 +168,18 @@ check_result(void)
     }
 }
 
+static void
+low_on_memory_event()
+{
+    /** Do nothing. Testing only register and unregister functions **/
```

Comment:
Remove the 2nd `*` (that makes it a Doxygen comment).
Also, style nit: missing ending `.`

Reply:
Done